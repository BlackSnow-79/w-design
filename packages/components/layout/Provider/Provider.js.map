{"version":3,"file":"Provider.js","sourceRoot":"","sources":["../../../../src/components/layout/Provider/Provider.tsx"],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;;;;;;;;;;AAAA,6BAA+B;AAC/B,gDAAsD;AACtD,+CAAiD;AACjD,+BAA0B;AAC1B,uCAAkC;AAMlC;IAAuB,4BAA0B;IAAjD;;IAiCA,CAAC;IAxBC,oCAAiB,GAAjB;QACE,IAAI,CAAC,UAAU,GAAG,QAAQ,CAAC,aAAa,CAAC,OAAO,CAAC,CAAC;QAClD,IAAI,CAAC,UAAU,CAAC,IAAI,GAAG,UAAU,CAAC;QAClC,IAAI,CAAC,UAAU,CAAC,SAAS,GAAG,kBAAQ,CAAC;QACrC,IAAI,QAAQ,IAAI,QAAQ,CAAC,IAAI,EAAE;YAC7B,QAAQ,CAAC,IAAI,CAAC,WAAW,CAAC,IAAI,CAAC,UAAU,CAAC,CAAC;SAC5C;IACH,CAAC;IAED,uCAAoB,GAApB;QACE,IAAI,IAAI,CAAC,UAAU,IAAI,QAAQ,IAAI,QAAQ,CAAC,IAAI,EAAE;YAChD,QAAQ,CAAC,IAAI,CAAC,WAAW,CAAC,IAAI,CAAC,UAAU,CAAC,CAAC;YAC3C,OAAO,IAAI,CAAC,UAAU,CAAC;SACxB;IACH,CAAC;IAED,yBAAM,GAAN;QACE,IAAM,eAAgC,EAA9B,gBAAK,EAAE,6BAAuB,CAAC;QACvC,OAAO,CACL,oBAAC,sBAAa,IAAC,KAAK,eAAO,eAAY,EAAK,KAAK;YAC/C,oBAAC,cAAI,eAAK,KAAY,EAAI,CACZ,CACjB,CAAC;IACJ,CAAC;IA7BM,oBAAW,GAAG,YAAY,CAAC;IAEpB,qBAAY,GAAU;QAClC,KAAK,EAAE,EAAE;KACV,CAAC;IA0BJ,eAAC;CAAA,AAjCD,CAAuB,KAAK,CAAC,aAAa,GAiCzC;AAED,kBAAe,QAAQ,CAAC"}