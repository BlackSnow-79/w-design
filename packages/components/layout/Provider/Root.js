"use strict";
var __makeTemplateObject = (this && this.__makeTemplateObject) || function (cooked, raw) {
    if (Object.defineProperty) { Object.defineProperty(cooked, "raw", { value: raw }); } else { cooked.raw = raw; }
    return cooked;
};
Object.defineProperty(exports, "__esModule", { value: true });
var styled_1 = require("../../../utils/styled");
var colors_1 = require("../../../common/colors");
exports.Root = styled_1.default.div(templateObject_1 || (templateObject_1 = __makeTemplateObject(["\n  font-family: ", ";\n\n  h1,\n  h2,\n  h3,\n  h4,\n  h5,\n  h6 {\n    font-weight: 500;\n    line-height: 1.25;\n    margin-bottom: 0.5em;\n    margin-top: 0;\n  }\n\n  h1,\n  h2,\n  h3,\n  h4 {\n    color: ", ";\n  }\n\n  h5,\n  h6 {\n    color: ", ";\n  }\n\n  p {\n    color: ", ";\n    line-height: 1.5;\n    margin-bottom: 0.5em;\n    margin-top: 0;\n  }\n\n  a {\n    color: ", ";\n    outline: none;\n    text-decoration: none;\n\n    &:hover {\n      color: ", ";\n    }\n\n    &:focus,\n    &:hover,\n    &:active {\n      text-decoration: underline;\n    }\n    &:focus,\n    &:active {\n      color: ", ";\n    }\n    &:focus {\n      text-decoration: none;\n    }\n  }\n\n  b,\n  strong {\n    font-weight: bolder;\n  }\n\n  code,\n  kbd,\n  pre,\n  samp {\n    font-family: \"Menlo, Courier, monospace\";\n  }\n\n  code {\n    color: ", ";\n    background: ", ";\n    padding: 0.1rem 0.15rem;\n    line-height: 1.2;\n  }\n"], ["\n  font-family: ", ";\n\n  h1,\n  h2,\n  h3,\n  h4,\n  h5,\n  h6 {\n    font-weight: 500;\n    line-height: 1.25;\n    margin-bottom: 0.5em;\n    margin-top: 0;\n  }\n\n  h1,\n  h2,\n  h3,\n  h4 {\n    color: ", ";\n  }\n\n  h5,\n  h6 {\n    color: ", ";\n  }\n\n  p {\n    color: ", ";\n    line-height: 1.5;\n    margin-bottom: 0.5em;\n    margin-top: 0;\n  }\n\n  a {\n    color: ", ";\n    outline: none;\n    text-decoration: none;\n\n    &:hover {\n      color: ", ";\n    }\n\n    &:focus,\n    &:hover,\n    &:active {\n      text-decoration: underline;\n    }\n    &:focus,\n    &:active {\n      color: ", ";\n    }\n    &:focus {\n      text-decoration: none;\n    }\n  }\n\n  b,\n  strong {\n    font-weight: bolder;\n  }\n\n  code,\n  kbd,\n  pre,\n  samp {\n    font-family: \"Menlo, Courier, monospace\";\n  }\n\n  code {\n    color: ", ";\n    background: ", ";\n    padding: 0.1rem 0.15rem;\n    line-height: 1.2;\n  }\n"])), function (props) { return props.theme.fonts; }, colors_1.colors.N90, colors_1.colors.N80, colors_1.colors.N70, colors_1.colors.B50, colors_1.colors.B30, colors_1.colors.B60, colors_1.colors.R20, colors_1.colors.N1);
exports.Root.displayName = "Root";
exports.default = exports.Root;
var templateObject_1;
//# sourceMappingURL=Root.js.map