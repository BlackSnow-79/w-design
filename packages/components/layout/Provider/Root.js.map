{"version":3,"file":"Root.js","sourceRoot":"","sources":["../../../../src/components/layout/Provider/Root.tsx"],"names":[],"mappings":";;;;;;AAAA,gDAA2C;AAC3C,iDAAgD;AAEnC,QAAA,IAAI,GAAG,gBAAM,CAAC,GAAG,++BAAA,mBACb,EAAiC,+LAkBrC,EAAU,sCAKV,EAAU,8BAIV,EAAU,oGAOV,EAAU,mFAKR,EAAU,+IAUV,EAAU,0OAoBZ,EAAU,qBACL,EAAS,+DAI1B,KA1EgB,UAAC,KAAU,IAAK,OAAA,KAAK,CAAC,KAAK,CAAC,KAAK,EAAjB,CAAiB,EAkBrC,eAAM,CAAC,GAAG,EAKV,eAAM,CAAC,GAAG,EAIV,eAAM,CAAC,GAAG,EAOV,eAAM,CAAC,GAAG,EAKR,eAAM,CAAC,GAAG,EAUV,eAAM,CAAC,GAAG,EAoBZ,eAAM,CAAC,GAAG,EACL,eAAM,CAAC,EAAE,EAIzB;AAEF,YAAI,CAAC,WAAW,GAAG,MAAM,CAAC;AAE1B,kBAAe,YAAI,CAAC"}