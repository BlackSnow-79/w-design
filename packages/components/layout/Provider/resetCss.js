"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var colors_1 = require("../../../common/colors");
/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
var ResetCss = "\nhtml {\n  line-height: 1.15;\n  -webkit-text-size-adjust: 100%;\n}\nbody {\n  margin: 0;\n}\nh1 {\n  font-size: 2em;\n  margin: 0.67em 0;\n}\nhr {\n  box-sizing: content-box;\n  height: 0;\n  overflow: visible;\n}\n\npre {\n  font-family: monospace, monospace;\n  font-size: 1em;\n}\n\na {\n  background-color: transparent;\n}\n\nabbr[title] {\n  border-bottom: none;\n  text-decoration: underline;\n  text-decoration: underline dotted;\n}\n\nb,\nstrong {\n  font-weight: bolder;\n}\n\ncode,\nkbd,\nsamp {\n  font-family: monospace, monospace;\n  font-size: 1em;\n}\n\n\nsmall {\n  font-size: 80%;\n}\n\nsub,\nsup {\n  font-size: 75%;\n  line-height: 0;\n  position: relative;\n  vertical-align: baseline;\n}\n\nsub {\n  bottom: -0.25em;\n}\n\nsup {\n  top: -0.5em;\n}\n\nimg {\n  border-style: none;\n}\n\n\nbutton,\ninput,\noptgroup,\nselect,\ntextarea {\n  font-family: inherit;\n  font-size: 100%;\n  line-height: 1.15;\n  margin: 0;\n}\n\n\nbutton,\ninput {\n  overflow: visible;\n}\n\n\n\nbutton,\nselect {\n  text-transform: none;\n}\n\n\nbutton,\n[type=\"button\"],\n[type=\"reset\"],\n[type=\"submit\"] {\n  -webkit-appearance: button;\n}\n\nbutton::-moz-focus-inner,\n[type=\"button\"]::-moz-focus-inner,\n[type=\"reset\"]::-moz-focus-inner,\n[type=\"submit\"]::-moz-focus-inner {\n  border-style: none;\n  padding: 0;\n}\n\nbutton:-moz-focusring,\n[type=\"button\"]:-moz-focusring,\n[type=\"reset\"]:-moz-focusring,\n[type=\"submit\"]:-moz-focusring {\n  outline: 1px dotted ButtonText;\n}\n\n\nfieldset {\n  padding: 0.35em 0.75em 0.625em;\n}\n\n\nlegend {\n  box-sizing: border-box;\n  color: inherit;\n  display: table;\n  max-width: 100%;\n  padding: 0;\n  white-space: normal;\n}\n\nprogress {\n  vertical-align: baseline;\n}\n\ntextarea {\n  overflow: auto;\n}\n\n\n[type=\"checkbox\"],\n[type=\"radio\"] {\n  box-sizing: border-box;\n  padding: 0;\n}\n\n\n[type=\"number\"]::-webkit-inner-spin-button,\n[type=\"number\"]::-webkit-outer-spin-button {\n  height: auto;\n}\n\n\n\n[type=\"search\"] {\n  -webkit-appearance: textfield;\n  outline-offset: -2px;\n}\n\n\n[type=\"search\"]::-webkit-search-decoration {\n  -webkit-appearance: none;\n}\n\n\n::-webkit-file-upload-button {\n  -webkit-appearance: button;\n  font: inherit;\n}\n\n\ndetails {\n  display: block;\n}\n\n\nsummary {\n  display: list-item;\n}\n\n\ntemplate {\n  display: none;\n}\n\n\n[hidden] {\n  display: none;\n}\n\n*,\n*::before,\n*::after {\n  box-sizing: inherit;\n}\n\n\nhtml {\n  box-sizing: border-box;\n  font-size: 20px;\n  line-height: 1.5;\n  -webkit-tap-highlight-color: transparent;\n}\n\nbody {\n  background: " + colors_1.colors.BLACK + ";\n  color: " + colors_1.colors.N60 + ";\n  font-family: '-apple-system, sans-serif, system-ui, BlinkMacSystemFont, Roboto, \"Helvetica Neue\"';\n  font-size: 0.8rem;\n  overflow-x: hidden;\n  text-rendering: optimizeLegibility;\n}\n\n";
exports.default = ResetCss;
//# sourceMappingURL=resetCss.js.map