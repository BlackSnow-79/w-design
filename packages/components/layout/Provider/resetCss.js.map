{"version":3,"file":"resetCss.js","sourceRoot":"","sources":["../../../../src/components/layout/Provider/resetCss.ts"],"names":[],"mappings":";;AAAA,iDAAgD;AAChD,4EAA4E;AAC5E,IAAM,QAAQ,GAAG,0jFA+MD,eAAM,CAAC,KAAK,oBACjB,eAAM,CAAC,GAAG,yMAOpB,CAAC;AAEF,kBAAe,QAAQ,CAAC"}