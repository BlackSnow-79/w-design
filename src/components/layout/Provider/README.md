Theme provider is based on [styled-components](https://www.styled-components.com/) and making theme with [styled-system](http://jxnblk.com/styled-system/)

Fallback style is based on normalize.css

```js
<Provider>
  <Container>
    <h1>test</h1>
    <p>
      Lorem ipsum
      <span>
        <a href="#">dolor </a>
      </span>
      sip amet
    </p>
  </Container>
</Provider>
```
