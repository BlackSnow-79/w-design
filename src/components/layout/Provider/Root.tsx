import styled from "../../../utils/styled";
import { colors } from "../../../common/colors";

export const Root = styled.div`
  font-family: ${(props: any) => props.theme.fonts};
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    color: ${colors.N90};
    font-weight: 500;
    line-height: 1.25;
    margin-bottom: 0.5em;
    margin-top: 0;
  }

  a {
    color: ${colors.B50};
    outline: none;
    text-decoration: none;

    &:hover {
      color: ${colors.B30};
    }

    &:focus,
    &:hover,
    &:active {
      text-decoration: underline;
    }
    &:focus,
    &:active {
      color: ${colors.B60};
    }
  }
`;

Root.displayName = "Root";

export default Root;
