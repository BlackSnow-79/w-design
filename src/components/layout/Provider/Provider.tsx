import * as React from "react";
import { ThemeProvider } from "../../../utils/styled";
import defaultTheme from "../../../common/theme";
import Root from "./Root";

export interface Props extends React.HTMLProps<HTMLDivElement> {
  theme?: any;
}

class Provider extends React.PureComponent<Props, void> {
  static displayName = "W.Provider";

  public static defaultProps: Props = {
    theme: {}
  };

  render() {
    const { theme, ...props } = this.props;

    return (
      <ThemeProvider theme={{ ...defaultTheme, ...theme }}>
        <Root {...props as any} />
      </ThemeProvider>
    );
  }
}

export default Provider;
