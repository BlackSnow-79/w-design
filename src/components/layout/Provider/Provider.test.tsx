// This component is based on styled-components, unnecessary to do overtest
import * as React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import Provider from "./Provider";

describe("<Provider/>", () => {
  // Render test
  it("should renders correctly", () => {
    const wrapper = shallow(<Provider />);
    expect(wrapper).toMatchSnapshot();
  });
});
