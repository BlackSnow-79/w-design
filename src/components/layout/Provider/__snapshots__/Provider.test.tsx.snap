// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Provider/> renders with custom theme 1`] = `
.c0 {
  font-family: Avenir Next;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6 {
  font-weight: 500;
  line-height: 1.25;
  margin-bottom: 0.5em;
  margin-top: 0;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4 {
  color: #10161A;
}

.c0 h5,
.c0 h6 {
  color: #182026;
}

.c0 p {
  color: #202B33;
  line-height: 1.5;
  margin-bottom: 0.5em;
  margin-top: 0;
}

.c0 a {
  color: #008CC0;
  outline: none;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 a:hover {
  color: #00A7E4;
}

.c0 a:focus,
.c0 a:hover,
.c0 a:active {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c0 a:focus,
.c0 a:active {
  color: #007099;
}

.c0 a:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 b,
.c0 strong {
  font-weight: bolder;
}

.c0 code,
.c0 kbd,
.c0 pre,
.c0 samp {
  font-family: "Menlo, Courier, monospace";
}

.c0 code {
  color: #F22828;
  background: #FBFBFB;
  padding: 0.1rem 0.15rem;
  line-height: 1.2;
}

<div
  className="c0"
/>
`;

exports[`<Provider/> should renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <W.Provider
    theme={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Root />,
      "theme": Object {
        "breakpoints": Array [
          "24em",
          "32em",
          "48em",
          "64em",
          "80em",
        ],
        "colors": Object {
          "B10": "#3ECBFF",
          "B20": "#00B7FA",
          "B30": "#00A7E4",
          "B40": "#009AD3",
          "B5": "#B1EAFF",
          "B50": "#008CC0",
          "B60": "#007099",
          "B7": "#7CDCFF",
          "B70": "#004660",
          "BLACK": "#000",
          "G10": "#79F2C0",
          "G20": "#57D9A3",
          "G30": "#36B37E",
          "G40": "#006644",
          "G5": "#E2FFEE",
          "G50": "#00875A",
          "G7": "#ABF5D1",
          "N1": "#FBFBFB",
          "N10": "#6C869B",
          "N2": "#EFF3F6",
          "N20": "#60798E",
          "N3": "#E2EAF2",
          "N30": "#566E82",
          "N4": "#CED9E3",
          "N40": "#4D6171",
          "N5": "#BECEDC",
          "N50": "#394B59",
          "N6": "#9DB3C6",
          "N60": "#293742",
          "N7": "#90A9BF",
          "N70": "#202B33",
          "N8": "#87A0B5",
          "N80": "#182026",
          "N9": "#768FA4",
          "N90": "#10161A",
          "O10": "#FFE380",
          "O20": "#FFC60A",
          "O30": "#FA9B00",
          "O40": "#EF8100",
          "O5": "#FFF7D7",
          "O50": "#E15B00",
          "O7": "#FFF19F",
          "P10": "#998DD9",
          "P20": "#8777D9",
          "P30": "#6554C0",
          "P40": "#5243AA",
          "P5": "#EAE6FF",
          "P50": "#403294",
          "P7": "#C0B6F2",
          "R10": "#FF5252",
          "R20": "#F22828",
          "R30": "#DC1F1F",
          "R40": "#C61515",
          "R5": "#FFADAD",
          "R50": "#B40404",
          "R7": "#FF8383",
          "T10": "#79E2F2",
          "T20": "#00C7E6",
          "T30": "#00B8D9",
          "T40": "#008DA6",
          "T5": "#E6FCFF",
          "T50": "#00A3BF",
          "T7": "#B3F5FF",
          "WHITE": "#FFF",
        },
        "fontSizes": Array [
          12,
          14,
          16,
          20,
          24,
          32,
          48,
          64,
          72,
          96,
        ],
        "fontWeights": Object {
          "bold": 700,
          "normal": 400,
          "xbold": 900,
        },
        "fonts": "-apple-system, BlinkMacSystemFont, Helvetica, \\"Helvetica Neue\\", system-ui, sans-serif",
        "letterSpacings": Object {
          "caps": "0.25em",
          "normal": "normal",
        },
        "lineHeights": Array [
          1,
          1.125,
          1.25,
          1.5,
        ],
        "space": Array [
          0,
          4,
          8,
          16,
          32,
          64,
          128,
        ],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Root />,
        "theme": Object {
          "breakpoints": Array [
            "24em",
            "32em",
            "48em",
            "64em",
            "80em",
          ],
          "colors": Object {
            "B10": "#3ECBFF",
            "B20": "#00B7FA",
            "B30": "#00A7E4",
            "B40": "#009AD3",
            "B5": "#B1EAFF",
            "B50": "#008CC0",
            "B60": "#007099",
            "B7": "#7CDCFF",
            "B70": "#004660",
            "BLACK": "#000",
            "G10": "#79F2C0",
            "G20": "#57D9A3",
            "G30": "#36B37E",
            "G40": "#006644",
            "G5": "#E2FFEE",
            "G50": "#00875A",
            "G7": "#ABF5D1",
            "N1": "#FBFBFB",
            "N10": "#6C869B",
            "N2": "#EFF3F6",
            "N20": "#60798E",
            "N3": "#E2EAF2",
            "N30": "#566E82",
            "N4": "#CED9E3",
            "N40": "#4D6171",
            "N5": "#BECEDC",
            "N50": "#394B59",
            "N6": "#9DB3C6",
            "N60": "#293742",
            "N7": "#90A9BF",
            "N70": "#202B33",
            "N8": "#87A0B5",
            "N80": "#182026",
            "N9": "#768FA4",
            "N90": "#10161A",
            "O10": "#FFE380",
            "O20": "#FFC60A",
            "O30": "#FA9B00",
            "O40": "#EF8100",
            "O5": "#FFF7D7",
            "O50": "#E15B00",
            "O7": "#FFF19F",
            "P10": "#998DD9",
            "P20": "#8777D9",
            "P30": "#6554C0",
            "P40": "#5243AA",
            "P5": "#EAE6FF",
            "P50": "#403294",
            "P7": "#C0B6F2",
            "R10": "#FF5252",
            "R20": "#F22828",
            "R30": "#DC1F1F",
            "R40": "#C61515",
            "R5": "#FFADAD",
            "R50": "#B40404",
            "R7": "#FF8383",
            "T10": "#79E2F2",
            "T20": "#00C7E6",
            "T30": "#00B8D9",
            "T40": "#008DA6",
            "T5": "#E6FCFF",
            "T50": "#00A3BF",
            "T7": "#B3F5FF",
            "WHITE": "#FFF",
          },
          "fontSizes": Array [
            12,
            14,
            16,
            20,
            24,
            32,
            48,
            64,
            72,
            96,
          ],
          "fontWeights": Object {
            "bold": 700,
            "normal": 400,
            "xbold": 900,
          },
          "fonts": "-apple-system, BlinkMacSystemFont, Helvetica, \\"Helvetica Neue\\", system-ui, sans-serif",
          "letterSpacings": Object {
            "caps": "0.25em",
            "normal": "normal",
          },
          "lineHeights": Array [
            1,
            1.125,
            1.25,
            1.5,
          ],
          "space": Array [
            0,
            4,
            8,
            16,
            32,
            64,
            128,
          ],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`<Provider/> should set stylesheet 1`] = `
<style
  type="text/css"
>
  
html {
  line-height: 1.15;
  -webkit-text-size-adjust: 100%;
}
body {
  margin: 0;
}
h1 {
  font-size: 2em;
  margin: 0.67em 0;
}
hr {
  box-sizing: content-box;
  height: 0;
  overflow: visible;
}

pre {
  font-family: monospace, monospace;
  font-size: 1em;
}

a {
  background-color: transparent;
}

abbr[title] {
  border-bottom: none;
  text-decoration: underline;
  text-decoration: underline dotted;
}

b,
strong {
  font-weight: bolder;
}

code,
kbd,
samp {
  font-family: monospace, monospace;
  font-size: 1em;
}


small {
  font-size: 80%;
}

sub,
sup {
  font-size: 75%;
  line-height: 0;
  position: relative;
  vertical-align: baseline;
}

sub {
  bottom: -0.25em;
}

sup {
  top: -0.5em;
}

img {
  border-style: none;
}


button,
input,
optgroup,
select,
textarea {
  font-family: inherit;
  font-size: 100%;
  line-height: 1.15;
  margin: 0;
}


button,
input {
  overflow: visible;
}



button,
select {
  text-transform: none;
}


button,
[type="button"],
[type="reset"],
[type="submit"] {
  -webkit-appearance: button;
}

button::-moz-focus-inner,
[type="button"]::-moz-focus-inner,
[type="reset"]::-moz-focus-inner,
[type="submit"]::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

button:-moz-focusring,
[type="button"]:-moz-focusring,
[type="reset"]:-moz-focusring,
[type="submit"]:-moz-focusring {
  outline: 1px dotted ButtonText;
}


fieldset {
  padding: 0.35em 0.75em 0.625em;
}


legend {
  box-sizing: border-box;
  color: inherit;
  display: table;
  max-width: 100%;
  padding: 0;
  white-space: normal;
}

progress {
  vertical-align: baseline;
}

textarea {
  overflow: auto;
}


[type="checkbox"],
[type="radio"] {
  box-sizing: border-box;
  padding: 0;
}


[type="number"]::-webkit-inner-spin-button,
[type="number"]::-webkit-outer-spin-button {
  height: auto;
}



[type="search"] {
  -webkit-appearance: textfield;
  outline-offset: -2px;
}


[type="search"]::-webkit-search-decoration {
  -webkit-appearance: none;
}


::-webkit-file-upload-button {
  -webkit-appearance: button;
  font: inherit;
}


details {
  display: block;
}


summary {
  display: list-item;
}


template {
  display: none;
}


[hidden] {
  display: none;
}

*,
*::before,
*::after {
  box-sizing: inherit;
}


html {
  box-sizing: border-box;
  font-size: 20px;
  line-height: 1.5;
  -webkit-tap-highlight-color: transparent;
}

body {
  background: #000;
  color: #293742;
  font-family: '-apple-system, sans-serif, system-ui, BlinkMacSystemFont, Roboto, "Helvetica Neue"';
  font-size: 0.8rem;
  overflow-x: hidden;
  text-rendering: optimizeLegibility;
}


</style>
`;
